specification/billing/resource-manager/Microsoft.Billing/stable/2024-04-01/billingAccount.json

```diff
@@ -0,0 +1,1342 @@
+      "properties": {
+        "status": {
+          "description": "Status of the address validation.",
+          "enum": [
+            "Other",
+            "Valid",
+            "Invalid"
+          ],
+          "type": "string",
+          "readOnly": true,
+          "x-ms-enum": {
+            "name": "AddressValidationStatus",
+            "modelAsString": true
+          }
+        },
+        "suggestedAddresses": {
+          "description": "The list of suggested addresses.",
+          "type": "array",
+          "items": {
+            "$ref": "./types.json#/definitions/AddressDetails"
+          },
+          "readOnly": true,
+          "x-ms-identifiers": []
+        },
+        "validationMessage": {
+          "description": "Validation error message.",
+          "type": "string",
+          "readOnly": true
+        }
+      },
+      "readOnly": true
+    },
+    "BillingAccount": {
+      "description": "A billing account.",
+      "type": "object",
+      "allOf": [
+        {
+          "$ref": "./types.json#/definitions/Resource"
+        }
+      ],
+      "properties": {
+        "properties": {
+          "description": "The properties of a(n) BillingAccount",
+          "allOf": [
+            {
+              "$ref": "#/definitions/BillingAccountProperties"
+            }
+          ],
+          "x-ms-client-flatten": true
+        }
+      },
+      "x-ms-azure-resource": true
+    },
+    "BillingAccountListResult": {
+      "description": "A container for a list of resources",
+      "type": "object",
+      "properties": {
+        "nextLink": {
+          "description": "The link (url) to the next page of results.",
+          "type": "string",
+          "readOnly": true
+        },
+        "value": {
+          "description": "The list of resources.",
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/BillingAccount"
+          },
+          "readOnly": true,
+          "x-ms-identifiers": [
+            "id"
+          ]
+        }
+      }
+    },
+    "BillingAccountPatch": {
+      "description": "A billing account.",
+      "type": "object",
+      "allOf": [
+        {
+          "$ref": "./types.json#/definitions/PatchResource"
+        }
+      ],
+      "properties": {
+        "properties": {
+          "description": "The properties of a(n) BillingAccount",
+          "allOf": [
+            {
+              "$ref": "#/definitions/BillingAccountProperties"
+            }
+          ],
+          "x-ms-client-flatten": true
+        }
+      },
+      "x-ms-azure-resource": true
+    },
+    "BillingAccountProperties": {
+      "description": "A billing account.",
+      "type": "object",
+      "properties": {
+        "provisioningState": {
+          "description": "The provisioning state of the resource during a long-running operation.",
+          "enum": [
+            "NotSpecified",
+            "Accepted",
+            "Running",
```

Comment:
>Running [](http://example.com/codeflow?start=13&length=7)

Why do you need both Running & Succeeded?

From your point of view, is this supposedly a terminal state, or a non-terminal one?
ARM needs it to be a non-terminal one, that will eventually transition to a terminal one, to avoid waiting forever any time it sees this. Just wondering if you actually have logic that would do that transition though.

But I have doubts you really need both